src/common/options.cc

```diff
@@ -429,6 +429,12 @@ std::vector<Option> get_global_options() {
     .set_flag(Option::FLAG_STARTUP)
     .add_service("common"),
 
+    Option("mon_host_override", Option::TYPE_STR, Option::LEVEL_DEV)
```

Comment:
we should probably add release notes/docs about this.

Reply:
It's really a testing config option, I don't think we'd ever have a reason for users to config this?

Reply:
I seem to recall there was a tracker issue from some user where this was helpful, but should only be needed for debugging purposes.

Reply:
If you think it's possible for users to want it, I'll bump it to LEVEL_ADVANCED and add a release note.

Reply:
done!